src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+.calendar {
+  padding: 30px;
+
+  &__container {
+    display: flex;
+    flex-wrap: wrap;
+    margin: 0 auto;
+    gap: 1px;
+    width: 7 * $square-side + 6 * $gap;
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    @include day-square ($square-side);
+
+    background-color: $bg-color;
+
+    font-size: 30px;
+
+    border: 1px solid $border-color;
+
+    transition: transform 0.5s ease, background-color 0.5s ease;
```

Comment:
Would be nice to create a variable for duration, then we can use it for every transition to make it's duration consistent with other page transitions